pkg/ansible/run.go

```diff
@@ -285,3 +283,20 @@ func getAnsibleDebugLog() bool {
 	}
 	return val
 }
+
+// getNamespacesForMetrics wil return all namespaces which will be used to export the metrics
```

Comment:
Is there a way we can centralize this logic? It seems like it's duplicated several times

Reply:
Move to the lib.